modules/common/pvc/pvc.go

```diff
@@ -80,7 +80,7 @@ func (p *Pvc) CreateOrPatch(
 	if err != nil {
 		if k8s_errors.IsNotFound(err) {
 			h.GetLogger().Info(fmt.Sprintf("Pvc %s not found, reconcile in %ds", pvc.Name, p.timeout))
-			return ctrl.Result{RequeueAfter: time.Duration(p.timeout) * time.Second}, nil
+			return ctrl.Result{RequeueAfter: p.timeout}, nil
```

Comment:
The above log line has `%ds` that will be wrong now. You can simply say `%s` and Duration will produce a human readable string.

Reply:
done